lib/utils/sanitization-utils.js

```diff
@@ -24,17 +24,35 @@ function getProtocol(url) {
     if (typeof url === 'string') {
       protocol = nodeURL.parse(url).protocol;
     }
-    return (protocol === null) ? ':' : protocol;
+    return protocol;
   } else {
-    throw new Error('Mobiledoc DOM Renderer is unable to sanitize href tags in this environment');
+    // unknown env where we cannot use a pre-existing url parser,
+    // such as Fastboot's VM Sandbox
+    return null;
   }
 }
 
-function sanitizeHref(url) {
+function urlStartsWithBadProtocol(url) {
```

Comment:
This function is basically a parser for HTTP protocol using `indexOf`, but `indexOf(badProtocol) ===0` isn't a valid parser for HTTP protocols on an href.

For example:

<img width="288" alt="screen shot 2017-03-01 at 1 48 41 pm" src="https://cloud.githubusercontent.com/assets/8752/23483004/f9ca06fa-fe85-11e6-904c-a8ab30fedbdb.png">

Meaning you could circumvent the bad protocol detection by prefixing the url string with spaces.